### PR TITLE
add contest blog in chinese

### PR DESCRIPTION
**Pull request by edisonxiang**
Add KubeEdge Contest Blog in Chinese

### CODE REVIEW

**Comment by edisonxiang** [comment]
cc @m1093782566 @kevin-wangzefeng @naveensriram @sids-b

**Comment by anvithks** [comment]
@edisonxiang send me the Chinese translation for `PARTICIPATE AND STAND A CHANCE 
TO WIN  TICKETS`.
I will create the featured image and sent it to you.

**Comment by edisonxiang** [comment]
@anvithks Thanks. The chinese content is like below:

参与就有机会赢得门票

**Comment by edisonxiang** [comment]
@anvithks Once you finish, please send the png to this following email. Thanks very much
80474298@qq.com

**Comment by anvithks** [comment]
> @anvithks Once you finish, please send the png to this following email. Thanks very much
> [80474298@qq.com](mailto:80474298@qq.com)

Sent it to you.
Cheers!

**Comment by edisonxiang** [comment]
The featured image was updated.
Thanks @anvithks @naveensriram

**Comment by edisonxiang** [comment]
This pr will depend on the pr #39.
The content will be sync with #39.
I will keep update.

**Comment by kevin-wangzefeng** [comment]
Let me know when it's ready for a final review.

File: content/zh/blog/contributor-contest/index.md [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+3. 在GitHub仓库[kubeedge](https://github.com/kubeedge/kubeedge) 
+/ [beehive](https://github.com/kubeedge/beehive) 
+/ [viaduct](https://github.com/kubeedge/viaduct) 
+/ [website](https://github.com/kubeedge/website)
+解决已经存在的Issue。
+
+4. 在GitHub仓库[kubeedge](https://github.com/kubeedge/kubeedge)提交Issue反馈用户需求。
+
+5. 在[KubeEdge website](https://kubeedge.io)或者其他博客网站发表关于KubeEdge的技术博客。
+请参考这个链接了解如何写一篇[KubeEdge博客](https://github.com/kubeedge/website/wiki/How-to-write-a-blog)。
```
Comment (kevin-wangzefeng) [inline content/zh/blog/contributor-contest/index.md]: 点击了解[如何在KubeEdge官网发表写一篇博客]

File: content/zh/blog/contributor-contest/index.md [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+请点击[这个链接](https://docs.google.com/forms/d/e/1FAIpQLSd9luGaHWt5jABEQjtr11CgWLVw7Px6tJf0TIjidbl3UL_Izg/viewform)提交您发表的博客信息。
```
Comment (kevin-wangzefeng) [inline content/zh/blog/contributor-contest/index.md]: 点击[反馈您发表的博客信息]

File: content/zh/blog/contributor-contest/index.md [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+
+{{% /alert %}}
+
+## **谁能参与？**
+
+任何人都可以！
+
+## **如何评选获胜者？**
+ 
+鉴于我们有各种各样的贡献方式，下面是我们对贡献的一些要求以及如何评选获胜者：
```
Comment (kevin-wangzefeng) [inline content/zh/blog/contributor-contest/index.md]: 鉴于本次活动接受多种贡献方式，下面是我们对贡献的一些要求以及如何产生获胜者：

File: content/zh/blog/contributor-contest/index.md [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+
+6. 在GitHub仓库[examples](https://github.com/kubeedge/examples)提交您对KubeEdge的使用样例和Demo。
+
+{{% /alert %}}
+
+## **谁能参与？**
+
+任何人都可以！
+
+## **如何评选获胜者？**
```
Comment (kevin-wangzefeng) [inline content/zh/blog/contributor-contest/index.md]: 如何产生获胜者

File: content/zh/blog/contributor-contest/index.md [hunk 5]
```diff
@@ -0,0 +1,144 @@
...
+
+- Issue提交：我们将考虑提交的Issue的严重性、复现步骤和数据的充分性。
+
+- 需求反馈：需求的唯一性，以及与其他边缘计算平台相比它可以带来的价值会作为评选的重要因素。
+
+- 项目推广：任何关于KubeEdge的博客、微信、Twitter、文章、白皮书等，我们会考虑它们的内容和受欢迎程度。
+
+- 使用样例：我们将考虑使用样例包含的代码、描述文档和用户指南等。
+
+**欢迎您以任何方式参与贡献，我们将会评选出3名获胜者!**
```
Comment (kevin-wangzefeng) [inline content/zh/blog/contributor-contest/index.md]: **欢迎您选择任意一种方式参与贡献，本次活动将产生3名获胜者。每位获胜者将获得2019中国上海KubeCon + CloudNativeCon & Open Source Summit 标准门票一张!**

**Comment by kevin-wangzefeng** [comment]
LGTM, thanks!

**Pull request merged**